src/main/resources/assets/create/lang/pl_pl.json

```diff
@@ -0,0 +1,1157 @@
+{
+
+	"_comment": "-------------------------]  GAME ELEMENTS  [------------------------------------------------",
+
+	"item.create.symmetry_wand": "Miernik symmetrii",
+	"item.create.placement_handgun": "Stawiacz bloków",
+	"item.create.terrain_zapper": "Rzeźbiarz świata",
+	"item.create.tree_fertilizer": "Nawóz do drzew",
+	"item.create.empty_blueprint": "Pusty schemat",
+	"item.create.andesite_alloy": "Stop andezytu",
+	"item.create.chromatic_compound": "Barwny związek",
+	"item.create.shadow_steel": "Cienista stal",
+	"item.create.blueprint_and_quill": "Schemat z piórem",
+	"item.create.blueprint": "Schemat",
+	"item.create.belt_connector": "Taśma mechaniczna",
+	"item.create.goggles": "Gogle inżynierskie",
+	"item.create.filter": "Filtr",
+	"item.create.property_filter": "Filtr atrybutów",
+	"item.create.rose_quartz": "Różowy kwarc",
+	"item.create.polished_rose_quartz": "Wypolerowany różowy kwarc",
+	"item.create.refined_radiance": "Rafinowany blaskowicz",
+	"item.create.iron_sheet": "Blaszka żelaza",
+	"item.create.gold_sheet": "Blaszka złota",
+	"item.create.lapis_plate": "Płytka lazurytu",
+	"item.create.obsidian_dust": "Sproszkowany obsidian",
+	"item.create.propeller": "Śmigło",
+	"item.create.whisk": "Trzepaczka",
+	"item.create.brass_hand": "Dłoń",
+	"item.create.slot_cover": "Klapka rzemieślnika mechanicznego",
+	"item.create.zinc_handle": "Cynkowa rękojeść",
```

Comment:
Although you are not wrong with the handle being made out of zinc, the English version is _"Quality Tool Handle"_.
Perhaps consider _"Rękojeść dobrej jakości"_?

Reply:
Is it indeed zinc handle or is it tool to check the quality of things?